doc/api/http.md

```diff
@@ -2213,6 +2213,8 @@ changes:
     **Default:** 8192 (8KB).
   * `method` {string} A string specifying the HTTP request method. **Default:**
     `'GET'`.
+  * `noDelay` {boolean} A boolean with which to call [`socket.setNoDelay()`][]
+    when a socket connection is established.
```

Comment:
Can you add a note that clarifies that this only works when the underlying connection is a network socket?

Reply:
Note added.